app/api/v1/auth/auth.py

```diff
@@ -42,18 +42,18 @@
 class ScopedTo(params.Depends):
     """Define a dependency on particular scope."""
 
+    username: str
```

Comment:
The call to `/user/me` wasn't working and in fixing it I needed to make this property's existence more explicit to avoid compiler errors.